lib/core/filtering/simple_rule.rb

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+#
+
+# frozen_string_literal: true
+
+module Core
+  module Filtering
+    class SimpleRule
+      POLICY = 'policy'
+      FIELD = 'field'
+      RULE = 'rule'
+      VALUE = 'value'
+      ID = 'id'
+
+      DEFAULT_RULE_ID = 'DEFAULT'
+      DEFAULT_RULE = SimpleRule.new(
+        {
+          'policy' => 'include',
+          'field' => '_',
+          'rule' => 'regex',
+          'value' => '.*',
+          'id' => SimpleRule::DEFAULT_RULE_ID
+        }
+      )
+
+      attr_reader :policy, :field, :rule, :value, :id
+
+      def initialize(rule_hash)
+        @policy = rule_hash.fetch(POLICY)
+        @field = rule_hash.fetch(FIELD)
+        @rule = rule_hash.fetch(RULE)
+        @value = rule_hash.fetch(VALUE)
+        @id = rule_hash.fetch(ID)
+      end
+
+      def match?(document)
+        return true if id == DEFAULT_RULE_ID
```

Comment:
I think it's not fool-proof. If one manipulates the data in `.elastic_connectors` one can assign a value `DEFAULT` to anything and therefore get inconsistent results. But I guess it's their problem if they tamper with data.

Reply:
> I guess it's their problem if they tamper with data

That's my feeling too. Crawler accomplished this by not actually _storing_ the default rule, so it couldn't get messed up. But that meant that you couldn't actually see what the rules were from looking at ES, which we need to be able to do now. We'll definitely need to rely more on Kibana to not allow breaking this ID or getting the ordering wrong, and hope that most people use the UI, or _carefully_ use programmatic access.